### ISSUE TITLE
node IteratorGetNext 

### ISSUE THREAD

**Issue by irfanahme** [rowanz/grover#21]
corrupted record at 0
         [[node IteratorGetNext (defined at /usr/local/lib/python3.5/dist-packages/tensorflow_estimator/python/estimator/util.py:110) ]]
 any one help me

**Comment by carlshi12r44** [comment]
same here

**Comment by irfanahme** [comment]
i solve this error one year ago

**Comment by irfanahme** [comment]
this error occur when you .tfrecord not create correctly

**Comment by carlshi12r44** [comment]
got it thanks but I not sure what the input is, I use a bunch of txt file, each file has some text content in it.

**Comment by irfanahme** [comment]
not txt create .tfrecord you need to use  .json file

**Comment by carlshi12r44** [comment]
oh gotcha thank you so much, does the Json file have any specific structure?

**Comment by irfanahme** [comment]
yes json have specific structure you check on sample i think one sample presents in code

**Comment by carlshi12r44** [comment]
gotcha thank you so much!
btw, when creating the tfrecords files, do we need to create some accounts on AWS and tried the `prepare_lm_data.py` in the `realnews` folder? Thanks again

**Comment by irfanahme** [comment]
yes
